lib/helpers.js

```diff
@@ -51,24 +51,11 @@ module.exports.getFirstWord = function (message) {
   return trimmed;
 };
 
-
 module.exports.isYesResponse = function (message) {
-  let trimmed = message.toUpperCase().trim();
-  trimmed = this.getFirstWord(trimmed);
-
-  const noResponses = [
-    'N',
-    'NA',
-    'NAH',
-    'NO',
-    'NIL',
-  ];
-
-  if (noResponses.indexOf(trimmed) >= 0) {
-    return false;
-  }
+  const trimmed = this.getFirstWord(message.toLowerCase().trim());
+  const yesResponses = process.env.GAMBIT_YES_RESPONSES ? `${process.env.GAMBIT_YES_RESPONSES}`.split(',') : ['yes'];
```

Comment:
Why do you need to convert `process.env.GAMBIT_YES_RESPONSES` to a string, isn't it a string already?